API Blueprint Specification.md

```diff
@@ -277,8 +280,12 @@ What sections can be nested and where depends upon the section in case, as descr
 # II. Sections Reference
 > **NOTE:** Sections marked as "Abstract" serve as a base for other sections and as such they **cannot** be used directly.
 
+<!-- BEGIN Abstract -->
+
+# Abstract
+
 <a name="def-named-section"></a>
-## 1. Named section
+## Named section
```

Comment:
My thought was that numbered lists like these are generally hard to maintain, and especially so in this case b/c the lists in the table of contents at the top would be automatically corrected by the Github Markdown parser in the case of order/number discrepancy, eg:

```
1. foo
3. bar
2. baz
```

will become:

1. foo
3. bar
2. baz

but the numbers within headers of sections would not be updated by the markdown parser.

If you want to keep consistency, I would suggest that we change the ordered lists in the table of contents at the top to be unordered lists (use "*"s instead of "#."s).

I will implement this change for the PR. Let me know if you would like something different.